lib/build.js

```diff
@@ -18,20 +16,28 @@ function build(options) {
             return stat.isDirectory();
         });
 
+    directories.forEach( (dir) => {
```

Comment:
This is/was basically a custom implementation of `mkdir -p`, which I think we can forgo in favour of `fs-extra`.